makepanda/makepandawrapper.py

```diff
@@ -0,0 +1,326 @@
+    custom_pkg_str = (
+        "  --<PKG>-incdir    (custom location for header files of thirdparty package)\n"
+        "  --<PKG>-libdir    (custom location for library files of thirdparty package)\n"
+    ) if sys.platform != 'win32' else ''
+    fail_str = 'makepanda: error: {}' if problem else ''
+    return (
+        "Makepanda generates a 'built' subdirectory containing a\n"
+        "compiled copy of Panda3D.  Command-line arguments are:\n"
+        "\n"
+        "  --help            (print the help message you're reading now)\n"
+        "  --verbose         (print out more information)\n"
+        "  --tests           (run the test suite)\n"
+        "  --installer       (build an installer)\n"
+        "  --wheel           (build a pip-installable .whl)\n"
+        "  --optimize X      (optimization level can be 1,2,3,4)\n"
+        "  --version X       (set the panda version number)\n"
+        "  --lzma            (use lzma compression when building Windows installer)\n"
+        "  --distributor X   (short string identifying the distributor of the build)\n"
+        "  --outputdir X     (use the specified directory instead of 'built')\n"
+        "  --threads N       (use the multithreaded build system. see manual)\n"
+        "  --osxtarget N     (the OS X version number to build for (OS X only))\n"
+        "  --override \"O=V\"  (override dtool_config/prc option value)\n"
+        "  --static          (builds libraries for static linking)\n"
+        "  --target X        (experimental cross-compilation (android only))\n"
+        "  --arch X          (target architecture for cross-compilation)\n"
+        + "".join([
+            "  --use-%-9s   --no-%-9s (enable/disable use of %s)\n" % (pkg.lower(), pkg.lower(), pkg)
+            for pkg in PKG_LIST
+        ]) +
+        custom_pkg_str +
+        "\n"
+        "  --nothing         (disable every third-party lib)\n"
+        "  --everything      (enable every third-party lib)\n"
+        "  --directx-sdk X   (specify version of DirectX SDK to use: jun2010, aug2009)\n"
+        "  --windows-sdk X   (specify Windows SDK version, eg. 7.1, 8.1 or 10.  Default is 8.1)\n"
+        "  --msvc-version X  (specify Visual C++ version, eg. 11, 12, 14, 14.1, 14.2.  Default is 14)\n"
+        "  --use-icl         (experimental setting to use an intel compiler instead of MSVC on Windows)\n"
+        "\n"
+        "The simplest way to compile panda is to just type:\n"
+        "\n"
+        "  makepanda --everything\n"
+        "\n" +
+        fail_str
+    )
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(usage=gen_usage(), prog='makepanda', add_help=False)
+    parser.add_argument('--help', '-h', action='store_true')
+    parser.add_argument('--verbose', action='store_true')
+    parser.add_argument('--tests', action='store_true')
+    # parser.add_argument('--installer', action='store_true')
+    # parser.add_argument('--wheel', action='store_true')
```

Comment:
CMake can't build wheels or packages; the idea here should be that we just run makewheel and makepackage on the CMake-generated built dir.

Reply:
CMake *could* be made to [create packages](https://cmake.org/cmake/help/latest/module/CPack.html); that might be worth setting up.

Wheels are an issue though. Maybe a virtual `wheel` target that just kicks makewheel would be in order.